src/main/scala/sigmastate/lang/SigmaBuilder.scala

```diff
@@ -187,6 +188,20 @@ trait SigmaBuilder {
   def mkPlusModQ(left: Value[SBigInt.type], right: Value[SBigInt.type]): Value[SBigInt.type]
   def mkMinusModQ(left: Value[SBigInt.type], right: Value[SBigInt.type]): Value[SBigInt.type]
 
+  def mkLogicalNot(input: Value[SBoolean.type]): Value[SBoolean.type]
+
+  def mkNegation[T <: SNumericType](input: Value[T]): Value[T]
+  def mkBitInversion[T <: SNumericType](input: Value[T]): Value[T]
+  def mkBitOr[T <: SNumericType](left: Value[T], right: Value[T]): Value[T]
+  def mkBitAnd[T <: SNumericType](left: Value[T], right: Value[T]): Value[T]
+  def mkBitXor[T <: SNumericType](left: Value[T], right: Value[T]): Value[T]
+  def mkBitShiftRight[T <: SNumericType](left: Value[T], right: Value[T]): Value[T]
+  def mkBitShiftLeft[T <: SNumericType](left: Value[T], right: Value[T]): Value[T]
+  def mkBitRotateLeft(left: Value[SCollection[SBoolean.type]],
+                      right: Value[SInt.type]): Value[SCollection[SBoolean.type]]
+  def mkBitRotateRight(left: Value[SCollection[SBoolean.type]],
+                      right: Value[SInt.type]): Value[SCollection[SBoolean.type]]
```

Comment:
@greenhat please make more meaningful arg names, like (bits, shift)

Reply:
Fixed in builder methods. IR nodes names remain the same due to `TwoArgumentsOperation` in the hierarchy.

Reply:
@greenhat I don't see much value in having this classification of operations. Maybe I missed something?

Reply:
Well, to generalize serializers we can add method `def args: Seq[SValue]` and have even more generic implementation.

Reply:
Agreed. Fixed.